interview.md

```diff
@@ -11,41 +11,94 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help
 them?_
 
+I interviewed a chemistry student who has taken multiple HMC engineering courses,
+and is currently in a chemistry focused engineering clinic project. As part of
+this project, his group is given programs that create visualizations of
+molecules. However, he expressed that it is confusing for chemists because
+the programs need to be compiled, and he thought that it would be easier for
+chemists to work with an interpreted programming language. He wanted the program
+to be as easy and intuitive for chemists with minimal CS background to work with.
+
 ### Why a language?
 
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+Yes, the domain is focused on a programming language that creates visualizations
+of molecules. It addresses the need by creating a program that is intuitive for
+chemists and helps them create visualizations of molecules.
+
 ### Why you?
 
 _What excites you about this idea? How did you come up with it?_
 
+I am exicted by this idea because it would make programming more accessible to
+scientists who themselves do not have a deep background in CS.
+I came up with this idea by interviewing a student, and asking him questions
+about why he and his teammates found the complication aspect of the molecule
+visualization programs confusing.
+
 ### Domain
 
 _Describe the project's domain in five words._
 
+Interpreted molecule visualization programming language.
+
 ### Interface (syntax)
 
 _How might the user interact with the language? What does programming look
 like? Why is this the right way to interact with the problem domain?_
 
+The user would be able to write in a coding language where they would define
+a molecules by defining atoms, bonds and the connections between the atoms and
+bonds. Then they could run the program and a visualization of the molecule would
+be created. Additionally, there could be information given about the molecule
+(maybe polarity, bond strength, etc.) after it is created. 
```

Comment:
I think the object-oriented approach you take here makes sense as chemistry is all about things being built out of smaller things. You could even view it as a graph kind of structure where each node is an atom and the edges are bonds.